src/main/java/org/apache/ibatis/jdbc/AbstractSQL.java

```diff
@@ -279,10 +281,12 @@ public T LIMIT(String variable) {
    *
    * @param value an offset value
    * @return a self instance
+   * @see #OFFSET(long)
    * @since 3.5.2
    */
   public T LIMIT(int value) {
     sql().limit = String.valueOf(value);
+    sql().limitingRowsStrategy = SQLStatement.LimitingRowsStrategy.OFFSET_LIMIT;
     return getSelf();
```

Comment:
The newly added line makes me think about DRY for some reason. :D
How about...

```java
return LIMIT(String.valueOf(value));
```

Reply:
@harawata Oh! good catch!!

Reply:
I've fixed via 953bba61f730f24cf6e81b7a98a058e50730e510.